Vagrantfile

```diff
@@ -55,18 +55,18 @@ Vagrant.configure("2") do |config|
     ./build.sh
 
 # Install opam and dependencies for compiling xapi
-    sudo apt-get install -y opam m4 libxen-dev
-    opam init -a --compiler=4.04.2 -y
-    eval `opam config env`
+    sudo apt-get install -y m4 libxen-dev
+    # Install opam 2:
+    sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)
+    opam init --disable-sandboxing --auto-setup --compiler=4.06.1 --yes
+    eval `opam env`
     opam remote add xs-opam git://github.com/xapi-project/xs-opam
     opam depext -y xapi
     opam install --deps-only xapi
 
 # Verify xapi can be built
     sudo apt-get install -y git
     git clone git://github.com/xapi-project/xen-api
-    # To avoid stack overflow error:
-    ulimit -s 16384
```

Comment:
Are you sure this is not needed anymore?

Reply:
Yep, Jon had a commit to split up xapi so that this isn't needed, we've removed it from other places